src/pygama/raw/build_raw.py

```diff
@@ -93,6 +102,8 @@ def build_raw(
                 in_stream_type = "FlashCam"
             elif OrcaStreamer.is_orca_stream(in_stream):
                 in_stream_type = "ORCA"
+            elif ext == "bin" or ext == "BIN":
```

Comment:
or if compass_config_file exists

Reply:
if the user sets a compass_config_file we don't need to check the extension, we can just assume it's a compass file.